core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -1008,6 +1009,10 @@ public void moveInProcessToError(String originalMessageId, String correlationId,
 						}
 					}
 				}
+				if(message == null && rawMessage instanceof Message) {
+					message = (Message) rawMessage;
+				}
+				throwEvent(RCV_MESSAGE_TO_ERRORSTORE_EVENT, message);
```

Comment:
If I'm reading the code right, this means that in theory the event could be thrown twice.

Reply:
Ja misschien dat hij niet gegooid moet worden wanneer je een errorSender gebruikt, maar niemand doet dat volgens mij?

Reply:
Geen idee of iemand dat doet, ergens.

Van mij mag het daarom gemerged worden zoals het is.